Code/Framework/AzFramework/Platform/iOS/AzFramework/Device/DeviceAttributesCommon_iOS.mm

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+ */
+
+#include <sys/utsname.h>
+#include <AzFramework/Device/DeviceAttributeDeviceModel.h>
+#include <AzFramework/Device/DeviceAttributeRAM.h>
+#include <AzCore/Console/ILogger.h>
+#include <Foundation/NSProcessInfo.h>
+
+
+namespace AzFramework
+{
+    DeviceAttributeDeviceModel::DeviceAttributeDeviceModel()
+    {
+        utsname systemInfo;
+        if (uname(&systemInfo) != -1)
+        {
+            m_value = systemInfo.machine;
+        }
+    }
+
+    DeviceAttributeRAM::DeviceAttributeRAM()
+    {
+        // the actual RAM amount available to the OS may be less
+        // that the physical RAM available, but is not readily available
+        constexpr double BytestoGiB = 1024.0 * 1024.0 * 1024.0;
+        auto memInBytes = [[NSProcessInfo processInfo] physicalMemory];
+        m_valueInGiB = aznumeric_cast<float>(static_cast<double>(memInBytes) / BytestoGiB);
+    }
+} // AzFramework

```

Comment:
Does the UnixLike implementation work for Mac or was that not tested?

Reply:
Unfortunately testing revealed UnixLike didn't work on Mac or iOS 😭

Reply:
Well does this implementation also work for Mac?
If so, this can be part of a `DeviceAttributesCommon_Apple.mm` file?

Reply:
good point, will test and merge if both work.